pylint_odoo/examples/.jslintrc

```diff
@@ -215,23 +223,23 @@
     "quote-props": "off",
     "quotes": "off",
     "radix": "error",
-    "require-jsdoc": "off",
+    "require-jsdoc": "warn",
```

Comment:
Can we detect which are the incorrect not documented instead of changing this to warn?

Reply:
The jsdoc parser is another option. This one raises a warn when there's a method without docstring. I don't wanna require it, but it's useful to have a reminder on the subject when you're writing a module.

Reply:
Then better to disable instead of having a lot of noise, but this is still useful in the case of new methods. That's why I ask if it's possible to distinguish them.

Reply:
I don't understand you... :sweat_smile: What do you want exactly and why?